pkg/codegen/python/importer.go

```diff
@@ -42,6 +42,8 @@ type PackageInfo struct {
 	Compatibility string `json:"compatibility,omitempty"`
 	// Indicates whether the package generates input/output classes.
 	UsesIOClasses bool `json:"usesIOClasses,omitempty"`
+	// Indicates whether the pulumiplugin.json file should be generated.
+	EmitPulumiPluginFile bool `json:"emitPulumiPluginFile,omitempty"`
```

Comment:
Any particular reason we don't want to do this by default? (Vs manually updating all providers to include this?)

Reply:
Ah, I recall now why I had this as an option. We're going to need to update the `Makefile` of each provider to replace `${PLUGIN_VERSION}` in `pulumiplugin.json` with the plugin version, just like we do with `setup.py`.

https://github.com/pulumi/pulumi-aws/blob/ed9cd60d861bc2d930150c272fa00fac2f774518/Makefile#L51